### PR TITLE
Add node descriptor to node status.

### PR DESCRIPTION
**Pull request by BramGruneir**

### CODE REVIEW

File: server/node.go [hunk 1]
```diff
@@ -452,6 +452,7 @@ func (n *Node) startStoresScanner(stopper *util.Stopper) {
 					StartedAt:  n.startedAt,
 					RangeCount: int32(rangeCount),
 					Stats:      *stats,
+					Desc:       n.Descriptor,
```
Comment (mrtracy) [inline server/node.go]: Do you need the NodeID separate on this structure now that you're including the full Desc?

**Comment by spencerkimball** [comment]
LGTM

File: server/node.go (hunk 1 above)
Comment (BramGruneir) [inline server/node.go]: Good call.  Done.

Comment (BramGruneir) [inline server/node.go]: Actually, I'll make a similar change to store status in another PR.

**Pull request merged**